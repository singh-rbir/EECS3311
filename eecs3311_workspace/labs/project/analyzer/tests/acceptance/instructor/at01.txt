-- Error: add expressions when an assignment instruction is not currently being specified

-- Expression 1 
addition

-- Expression 2: empty call chain (which has lower priority in the message table; see instructions)
add_call_chain(<<>>)

-- Expression 3: call chain a
add_call_chain(<<"a">>)

-- Expression 4: call chain a.b
add_call_chain(<<"a", "b">>)

-- And similarly for all other expression-related events.