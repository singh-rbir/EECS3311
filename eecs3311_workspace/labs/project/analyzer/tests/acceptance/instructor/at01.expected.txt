  Status: OK.
  Number of classes being specified: 0
->addition
  Status: Error (An assignment instruction is not currently being specified).
  Number of classes being specified: 0
->add_call_chain(<<>>)
  Status: Error (An assignment instruction is not currently being specified).
  Number of classes being specified: 0
->add_call_chain(<<"a">>)
  Status: Error (An assignment instruction is not currently being specified).
  Number of classes being specified: 0
->add_call_chain(<<"a", "b">>)
  Status: Error (An assignment instruction is not currently being specified).
  Number of classes being specified: 0
